webpack.config.prod.js

```diff
@@ -1,23 +1,35 @@
 const path = require('path');
 const HWP = require('html-webpack-plugin');
-module.exports = {
+module.exports = {  
     mode: 'production',
     entry: path.join(__dirname, '/src/index.js'),
     output: {
        filename: 'build.js',
-       path: path.join(__dirname, '/prod/dist')},
+       path: path.join(__dirname, '/dev/dist')},
```

Comment:
It is strange a little to use the same folder **dev** for the dev and production apps. Please, return **/prod/dist** for the prod build configuration